charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -238,6 +253,44 @@ spec:
           mountPath: /etc/kubernetes/etcd-encryption-secret
           readOnly: true
         {{- end }}
+      {{- if .Values.konnectivityTunnel.enabled }}
+      - name: konnectivity-server
+        image: {{ index .Values.images "konnectivity-server" }}
+        command:
+          - /proxy-server
+        args:
+          - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
+          - --logtostderr=true
+          - --cluster-cert=/certs/konnectivity-server/konnectivity-server.crt
+          - --cluster-key=/certs/konnectivity-server/konnectivity-server.key
+          - --agent-namespace={{ .Values.konnectivityTunnel.agentNamespace }}
+          - --agent-service-account=konnectivity-agent
+          - --kubeconfig=/etc/srv/kubernetes/konnectivity-server-kubeconfig/kubeconfig
+          - --authentication-audience=system:konnectivity-server
+          - --mode=http-connect
+          - --server-count={{ .Values.konnectivityTunnel.serverCount }}
+          - --server-port={{ .Values.konnectivityTunnel.serverPort }}
+          - --agent-port={{ .Values.konnectivityTunnel.agentPort }}
+          - --admin-port={{ .Values.konnectivityTunnel.adminPort }}
+        resources:
+{{ toYaml .Values.konnectivityTunnelResources | indent 10 }}
+        ports:
+          - name: agentport
+            containerPort: {{ .Values.konnectivityTunnel.agentPort }}
+            hostPort: {{ .Values.konnectivityTunnel.agentPort }}
+          - name: adminport
+            containerPort: {{ .Values.konnectivityTunnel.adminPort }}
+            hostPort: {{ .Values.konnectivityTunnel.adminPort }}
+        volumeMounts:
+          - name: konnectivity-server-certs
+            mountPath: /certs/konnectivity-server
+            readOnly: true
+          - name: konnectivity-server-kubeconfig
+            mountPath: /etc/srv/kubernetes/konnectivity-server-kubeconfig
+          - name: konnectivity-uds
+            mountPath: /etc/srv/kubernetes/konnectivity-server
+            readOnly: false
```

Comment:
can you indent all items in the list to match the rest of the file?

Reply:
I am not sure I follow what items you mean? the indentation works, otherwise I will break the deployment.

Reply:
Instead of 

```yaml
        args:
          - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
          - --logtostderr=true
```

to have something like:

```yaml
        args:
        - --uds-name=/etc/srv/kubernetes/konnectivity-server/konnectivity-server.socket
        - --logtostderr=true
```